standard/links_stream.lua

```diff
@@ -115,14 +143,21 @@ end
 ---@field languageCode string
 ---@field index integer
 ---@field is_a function
+---@field new function
+---@field _fromLegacy function
+---@field toLegacy function
+---@field toString function
+---@field _isStreamKey function
+---@field _isValid function
+---@field __tostring function
```

Comment:
The functions shouldn't be needed to be specified in the class

Reply:
if defining the class above `StreamLinks.StreamKey` they are needed
if switching it to local and setting `StreamLinks.StreamKey = StreamKey` it works without